### PR TITLE
updates to v1.2.0 changelog

### PR DESCRIPTION
**Pull request by hoozecn**
#### What type of this PR

Add one of the following kinds:
/kind document

#### What this PR does / why we need it:

updates to v1.2.0 changelog

#### Which issue(s) this PR fixes:

- NaN

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     updates to v1.2.0 changelog     |
| 🇨🇳 中文    |     v1.2.0 变更记录修正          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by daskyrk** [comment]
Maybe can fix the link problem by the way.
image

**Comment by hoozecn** [comment]
> Maybe can fix the link problem by the way.
> image

nice catch

File: CHANGELOG/CHANGELOG-1.x.md [hunk 1]
```diff
@@ -36,6 +36,8 @@
 * Optimize pipeline log style. [#802](https://github.com/erda-project/erda-ui/pull/802)
 * Optimize pipeline notification content. [#1189](https://github.com/erda-project/erda/pull/1189)
 * Optimize the execution logic of automation test cases. [#1103](https://github.com/erda-project/erda/pull/1103)
+* Support filtering test case executor by unassigned person in the test plan. [#732](https://github.com/erda-project/erda/pull/732)
+* Add precheck for front-end applicaiton packaging.
```
Comment (daskyrk) [inline CHANGELOG/CHANGELOG-1.x.md:40]: Suggest to describe more clear as `precheck ths package-lock.json file's existence`

Comment (hoozecn) [reply CHANGELOG/CHANGELOG-1.x.md:40]: Yeah it's more accurate

Comment (hoozecn) [reply CHANGELOG/CHANGELOG-1.x.md:40]: @daskyrk Do you know what the PR refer to?

Comment (daskyrk) [reply CHANGELOG/CHANGELOG-1.x.md:40]: Nope, it'a not in frontend repo.

Comment (daskyrk) [reply CHANGELOG/CHANGELOG-1.x.md:40]: Suggest to search in erda issues and find out who is responsible for this job.

**Pull request merged**